egs/wsj/s5/steps/nnet3/tdnn/make_configs.py

```diff
@@ -102,7 +102,11 @@ def GetArgs():
     parser.add_argument("--pnorm-output-dim", type=int,
                         help="output dimension of p-norm nonlinearities")
     parser.add_argument("--relu-dim", type=int,
```

Comment:
It is a bit confusing to see the same option being used in two different ways, based on specification of other options. I would rather use the `parser.add_mutually_exclusive_group`  option to specify either `--relu-dim` or `--relu-final-dim` options.